dockers/docker-sonic-telemetry/telemetry.sh

```diff
@@ -19,7 +19,7 @@ else
 	TELEMETRY_ARGS+=" --insecure"
 fi
 
-if [ ! -z $X509 ]; then
+if [ ! -z "$X509" ]; then
```

Comment:
Also suggest simplifying. `! -z` == `-n`, thus,

```suggestion
if [ -n "$X509" ]; then
```